src/sentry/static/sentry/app/components/events/interfaces/spans/inlineDocs.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import * as Sentry from '@sentry/browser';
+
+import withApi from 'app/utils/withApi';
+import {Client} from 'app/api';
+import {loadDocs} from 'app/actionCreators/projects';
+import {t, tct} from 'app/locale';
+import LoadingIndicator from 'app/components/loadingIndicator';
+
+type Props = {
+  api: Client;
+
```

Comment:
Don't need the newline here.

Reply:
Shouldn't this just get caught by prettier?